### PR TITLE
fixes

### LINKED ISSUE #5851: (WEB REPORT BY: talux00 REMOTE: 198.245.63.50:7777) Bug
Revision: 8863d47208e80bec8cb19bc09ea9936e673823e2 

>General description of the issue: When you use an evidence bag on a fire extinguisher it "get stuck in the muzzle" but when you use the fire extinguisher you fire the evidence bag with a fire extinguisher inside.

>What you expected to happen

>What actually happened

>Steps to reproduce if possible

### LINKED ISSUE #5856: (WEB REPORT BY: boptser REMOTE: 198.245.63.50:7777) Security and crew monitoring computer while in a wheelchair
The basic issue is that while in a wheelchair, the security cameras computer doesn't seem to display anything after selecting the camera 

This is not the case with the crew monitoring monitor, that displays everything when clicked 

the issue reverses when you're no longer on the wheelchair and without legs, the security camera PC works and the crew monitor doesn't

this is clearly a gamebreaking issue and without it being fixed this whole station will fall apart

### LINKED ISSUE #5862: You can talk in a split second before cloning begins
Consciousness slowly creeps over you as your body regenerates.
So this is what cloning feels like?
[DATA EXPUNGED] exclaims, "[REDACTED]"
You can almost hear someone talking.

Basically, you could communicate with people the second before cloning crits you.

### PR DESCRIPTION
**Pull request by unid15**
- security camera computers may now be used while in a wheelchair (Fixes #5856)
- evidence bags no longer fit into extinguishers (Fixes #5851)
- shuttle initialize() proc now correctly logs errors when initializing
- shuttle turfs moved from turfs/simulated/floor_types.dm to turfs/simulated/shuttle.dm 
- removed some secrets which no longer work
- clones grown in the clone pod are silenced for a few seconds after creation (bandaid Fixes #5862)

### CODE REVIEW

File: code/modules/medical/cloning.dm [hunk 1]
```diff
@@ -229,6 +229,7 @@
 	H.adjustCloneLoss(150) //new damage var so you can't eject a clone early then stab them to abuse the current damage system --NeoFite
 	H.adjustBrainLoss(src.heal_level + 50 + rand(10, 30)) // The rand(10, 30) will come out as extra brain damage
 	H.Paralyse(4)
+	H.silent += 10 //Can't talk for a while
```
Comment (clusterfack) [inline code/modules/medical/cloning.dm]: This could probably also be fixed by setting stat = 1 so that they are unconscious immediately

Comment (dylanstrategie) [inline code/modules/medical/cloning.dm]: >stat = 1

Jesus Christ clusterfack

Comment (clusterfack) [inline code/modules/medical/cloning.dm]: we all know what it means when I say that fag

Comment (d3athrow) [inline code/modules/medical/cloning.dm]: coulda swore i told him to set stat manually when he asked a few days ago

**Pull request merged**